packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -156,12 +157,9 @@ export class GreasedLinePluginMaterial extends MaterialPluginBase {
 
     private static _EmptyColorsTexture: BaseTexture;
 
-    constructor(
-        material: Material,
-        private _scene: Scene,
+    constructor(material: Material, private _scene: Scene, options: GreasedLineMaterialOptions) {
+        options = options ?? {};

```

Comment:
It's not the case. When creating a new GreasedLinePluginMaterial, you must provide this parameter, it's not optional. The code `options = options ?? {}` has been added because the way the plugin cloning is implemented.
[I wrote about it here](https://forum.babylonjs.com/t/material-clone-doesnt-clone-all-material-plugins/41695/3?u=roland). When you `Instantiate` it, it will provide call the constructor without the `options` parameter.